src/public/target/sections/GeneOntology/Section.js

```diff
@@ -2,8 +2,20 @@ import React from 'react';
 
 import OntologyTable from './custom/OntologyTable';
 
-const Section = ({ symbol, ensgId, data }) => (
-  <OntologyTable rows={data.rows} symbol={symbol} />
+const CATEGORY_BY_PREFIX = {
+  P: 'BIOLOGICAL_PROCESS',
+  F: 'MOLECULAR_FUNCTION',
+  C: 'CELLULAR_COMPONENT',
+};
+
+const extractCategory = row => ({
+  ...row,
+  category: CATEGORY_BY_PREFIX[row.term.substring(0, 1)],
+  term: row.term.substring(2),
+});
```

Comment:
Let me know if you think this function is unreadable without more documentation on what the input looks like. I considered listing examples in unit tests, but I don't think there's a suite to run them and ensure they stay in sync with the code.

Reply:
Thank you Fedde - I think it's OK to understand as it's also quite short.
I don't mind either way: a short `e.g. term format: "F:ATPase binding"` could help as it's very specific to the data returned by the API (I'm a bit old school in regards to code comments). Even in the current angular app we do things that assume the data to be in a certain format and I'm sure they're not always documented...